src/replication.c

```diff
@@ -933,6 +1003,7 @@ send_replica_handshake_query(replica_t *replica, spec_t *spec)
 	int ret = 0;
 
 	mgmt_cmd = malloc(sizeof(mgmt_cmd_t));
+	memset(mgmt_cmd, 0, sizeof (mgmt_cmd_t));
```

Comment:
Once we zero out this memory, we do not need code at line 1016 and 1019. We can remove those lines.